sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -50,57 +50,110 @@
         this.service = service;
     }
 
-    Mono<Response<DocumentResultCollection<AnalyzeSentimentResult>>> analyzeSentimentBatchWithResponse(
+    /**
+     * Helper function that analyzes a batch of documents and returns {@link TextAnalyticsPagedFlux} that is a paged
+     * flux contains {@link AnalyzeSentimentResult}.
+     *
+     * @param textInputs A batch of input texts.
+     * @param options The request options, such as the training model version and to show statistics.
+     *
+     * @return {@link TextAnalyticsPagedFlux} of {@link AnalyzeSentimentResult}.
+     */
+    TextAnalyticsPagedFlux<AnalyzeSentimentResult> analyzeSentimentBatch(Iterable<TextDocumentInput> textInputs,
+        TextAnalyticsRequestOptions options) {
+        Objects.requireNonNull(textInputs, "'textInputs' cannot be null.");
+        try {
+            return new TextAnalyticsPagedFlux<>(() -> (continuationToken, pageSize) -> withContext(context ->
+                service.sentimentWithRestResponseAsync(
+                    new MultiLanguageBatchInput().setDocuments(Transforms.toMultiLanguageInput(textInputs)),
+                    options == null ? null : options.getModelVersion(),
+                    options == null ? null : options.isStatisticsShown(), context)
+                    .doOnSubscribe(ignoredValue ->
+                        logger.info("A batch of text sentiment input - {}", textInputs.toString()))
+                    .doOnSuccess(response -> logger.info("A batch of text sentiment output - {}", response))
+                    .doOnError(error -> logger.warning("Failed to analyze text sentiment - {}", error))
+                    .map(this::toTextAnalyticsPagedResponse))
+                    .flux());
+        } catch (RuntimeException ex) {
+            return new TextAnalyticsPagedFlux<>(() ->
+                (continuationToken, pageSize) -> fluxError(logger, ex));
+        }
+    }
+
+    /**
+     * Helper function that calling service with max overloaded parameters and returns {@link TextAnalyticsPagedFlux}
+     * that is a paged flux contains {@link AnalyzeSentimentResult}.
```

Comment:
```suggestion
     * Helper function for calling service with max overloaded parameters that a returns {@link TextAnalyticsPagedFlux}
     * which is a paged flux that contains {@link AnalyzeSentimentResult}.
```
or something like that